site/pages/components/radio-button.tsx

```diff
@@ -161,15 +186,638 @@ const RadioButtonComponent = (layoutProps: LayoutProps) => (
       },
       introduction: `Radio Buttons are selection controls that are typically used in forms. They are used for exclusive selection - allowing users to select one of multiple options in a Radio Group.`,
     }}
-    componentDefaultsKey="select"
+    componentDefaultsKey="radio"
     meta={{
-      status: MetaStatus.Beta,
-      introduced: 'v5.3',
+      status: MetaStatus.Supported,
```

Comment:
We can add it as Beta for now
```suggestion
      status: MetaStatus.Beta,
```

Reply:
Sure Stoyan, updated the suggested changes. Thanks.